pom.xml

```diff
@@ -300,5 +302,61 @@
        </plugins>
       </build>
     </profile>
+    <profile>
+      <id>get-gcloud-project-id</id>
+      <activation>
+        <property>
+          <name>!app.deploy.project</name>
+        </property>
+      </activation>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-antrun-plugin</artifactId>
+            <version>1.8</version>
+            <executions>
+              <execution>
+                <id>find-app-deploy-project</id>
```

Comment:
This acceptable?

https://github.com/joakime/jetty-runtime/blob/3dd892bcb126bf2bddf83177c505c7a119147e15/pom.xml#L306-L309